src/main/java/bio/terra/common/logging/GoogleJsonLayout.java

```diff
@@ -63,8 +62,6 @@ class GoogleJsonLayout extends JsonLayoutBase<ILoggingEvent> {
   // A reference to the current Spring app context, on order to pull out the spring.application.name
   // and spring.application.version variable for inclusion in JSON output.
   private ConfigurableApplicationContext applicationContext;
-  // A Gson instance to support converting Gson-type payloads into Jackson nodes.
```

Comment:
So we're still using Gson, just not this instance?

Reply:
Yep. This still supports Gson (and `toJsonMap` still handles `gson.JsonObject`s), this is just cleaning up an unused field.